concent_api/core/utils.py

```diff
@@ -118,3 +123,34 @@ def extract_name_from_scene_file_path(absoulte_scene_file_path_in_docker: str) -
             ErrorCode.MESSAGE_INVALID
         )
     return relative_scene_file_path_in_archive
+
+
+def is_protocol_verison_compatible_with_verison_in_concent(protocol_version: str) -> bool:
+    """
+    Versions are considered compatible if they share the minor and major version number.
+    E.g 2.18.5 is compatible with 2.18.1 but not with 2.17.5 or 3.0.0.
+    """
+    return protocol_version.split('.')[0] == settings.GOLEM_MESSAGES_VERSION.split('.')[0] and \
```

Comment:
I think regex should be used here. BTW: Are we sure that 
`GOLEM_MESSAGES_VERSION` is always in `A.B.C` format?

Reply:
Or maybe somehow with `startswith`? Either way, styling should be fixed.

Reply:
I think so, it always was like this and example in blueprint is the same. Whats your ideas to fix it? Now there are two easy to read lines- how to do it better/nicer/in better style?

Reply:
you are overusing split. have it like:
```python
major, minor, _ = protocol_version.split('.')
concent_majnor, concent_minor, _ = settings.GOLEM_MESSAGES_VERSION.split('.')
return major == concent_major and minor == concent_minor
```

Reply:
changed